components/WrapPanel2/samples/WrapPanel2.md

```diff
@@ -0,0 +1,54 @@
+---
+title: WrapPannel2
+author: Avid29
+description: A labs-component candidate for a new WrapPanel implementation.
+keywords: WrapPanel, Control, Layout
+dev_langs:
+  - csharp
+category: Layouts
+subcategory: Panel
+discussion-id: 762
+issue-id: 763
+icon: assets/icon.png
+---
+
+# WrapPanel2
+
+The WrapPanel2 is an experiment for a new WrapPanel API using GridLength definitions to define the item's desired sizings.
+
+When stretched along the main axis, the child elements with star-sized GridLength values will proportionally occupy the available space.
+
+When not stretched along the main axis, star-sized child elements will be the smallest size possible while maintaining proportional sizing relative to each other and ensuring that all child elements are fully visible.
+
+
+> [!Sample WrapPanel2BasicSample]
```

Comment:
Thinking it could be good to also split out some more dedicated smaller/fixed examples that show interesting/dedicated uses. The uber example is fun to play with, but I think makes it hard to understand where to use each type of functionality.

i.e. the idea is that the panel is flexible to use in a variety of scenarios, but it'd be rare to try and use every aspect at once.